app/models/list_account.rb

```diff
@@ -30,4 +31,12 @@ def set_follow
   rescue ActiveRecord::RecordNotFound
     self.follow_request = FollowRequest.find_by!(account_id: list.account_id, target_account_id: account.id)
   end
+
+  def validate_relationship
```

Comment:
I realize this is 18 months old, but I'm curious on the logic here ... I had added some coverage for the callback portion of this (set_follow method, above) and was going to do same for the validations -- but I don't understand the path where these validations are actually run.

When `list.account_id == account_id` both the `set_follow` callback and this `validation_relationship` will bail out. In other scenarios, the `set_follow` will run first and (it seems to me?) either set values or raise in a way where I don't see how this method would find any of these scenarios.

Or - I see what these validations are and why we have them, but it seems either the first method would success in finding/setting (in which case this will not find errors), or it will raise while running (in which case this will never run).

I feel like I'm missing some edge case here that would let me create the path to exercise this.